### DISCUSSION TITLE
[question] Fintunes audio quality

### DISCUSSION THREAD

**Discussion by Mis1ify** [leinelissen/jellyfin-audio-player#132] [Q&A]
What's the audio quality of music? If I have FLAC, does the app limit streamed quality to something worse?

**Comment by leinelissen** [comment]
Hey there! I've converted your issue to a discussion so I don't have to close it. The answer depends on the source container and on which OS you're using. On both Android and iOS, FLAC will be played back directly, with iOS also featuring support for ALAC. Support for transcoding to lower bitrates on-the-fly (to save your data plan) is in the works (and tracked in #118) but is blocked by some work that needs to happen in Jellyfin core (see details in #97). 

I will try and add a supported codecs table to the README and eventual website when I find some time.

**Comment by Mis1ify** [comment]
Thx for answer!
